### PR TITLE
Verify that source files contains a valid copyright header

### PR DESCRIPTION
**Pull request by mwperina**
Signed-off-by: Martin Perina <mperina@redhat.com>

### CODE REVIEW

File: selinux/meson.build [hunk 1]
```diff
@@ -1,4 +1,7 @@
+#
 # Copyright 2022 Collabora Ltd.
+# Copyright Contributors to the Eclipse BlueChi project
```
Comment (engelmi) [inline selinux/meson.build]: This file has already a Copyright. Is it ok to add it for BlueChi here as well?

**Review by engelmi — APPROVED** [review]
Small question, otherwise LGTM

File: selinux/meson.build (hunk 1 above)
Comment (mwperina) [reply selinux/meson.build]: We have adapted the original file from https://github.com/flatpak/flatpak/blob/main/selinux/meson.build so it seems OK to me add our own Copyright. The other option would be to leave original copyright as is and not check the file within the script.

@alexlarsson What do you think?

Comment (mwperina) [reply selinux/meson.build]: Leaving original copyright for now

**Pull request merged**